clusters/app.ci/release-controller/deploy-ocp-arm64-priv-controller.yaml

```diff
@@ -0,0 +1,439 @@
+##################################################################################
+#                                DO NOT EDIT
+# File generated during execution of: generate-release-controllers.py
+##################################################################################
+
+
+#---------------------------------------------------------------------------------
+# osd_rc_deployments.py added the following resource
+#---------------------------------------------------------------------------------
+
+# Resources required to deploy release controller instances on
+# the app.ci clusters.
+# Bootstrap the environment for the amd64 tests image.  The caches require an amd64 "tests" image to execute on
+#     the cluster.  This imagestream is used as a commandline parameter to the release-controller...
+#          --tools-image-stream-tag=release-controller-bootstrap:tests
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: release-controller-bootstrap
+  namespace: ocp-arm64-priv
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - from:
+      kind: DockerImage
+      name: image-registry.openshift-image-registry.svc:5000/ocp/4.6:tests
```

Comment:
`4.8` ?

Reply:
All the release-controllers utilize the same `tests` image.  We can update if there is a specific need, but it's not a requirement.